src/Misc/layoutbin/powershell/Add-DotNetFrameworkCapabilities.ps1

```diff
@@ -94,7 +94,8 @@ function Add-Versions {
                 { $_ -gt 379893 -and $_ -le 380995 } { "4.5.3" }
                 { $_ -gt 380995 -and $_ -le 393297 } { "4.6.0" }
                 { $_ -gt 393297 -and $_ -le 394271 } { "4.6.1" }
-                { $_ -gt 394271 } { "4.6.2" }
+                { $_ -gt 394271 -and $_ -le 460797 } { "4.6.2" }
```

Comment:
base on [this](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed)
I think it should be:
```ps1
{ $_ -gt 394271 -and $_ -le 394806 } { "4.6.2" }
{ $_ -gt 394806 } { "4.7.0" }
```

You might want to change https://github.com/Microsoft/vsts-agent/blob/2e78d84ad10e2a71ad0a4188c2ca38a106de2bc4/src/Microsoft.VisualStudio.Services.Agent/Util/NetFrameworkUtil.cs#L157 as well

Reply:
Thank you for review and recommendations!